src/cascadia/UnitTests_Control/MockControlSettings.h

```diff
@@ -52,7 +52,7 @@ namespace ControlUnitTests
         WINRT_PROPERTY(winrt::hstring, FontFace, L"Consolas");
         WINRT_PROPERTY(int32_t, FontSize, DEFAULT_FONT_SIZE);
 
-        WINRT_PROPERTY(winrt::Windows::UI::Text::FontWeight, FontWeight);
+        WINRT_PROPERTY(winrt::Windows::UI::Text::FontWeight, FontWeight, 400);

```

Comment:
Does this change imply that we have a divide-by-zero if the user configures a font weight of 0?

Reply:
No, the behaviour of part of the code actually changed, so it can't handle the case. I'm still working on it.